Makefile

```diff
@@ -1,8 +1,18 @@
 IMAGE_NAME=quay.io/theforeman/candlepin
 IMAGE_TAG=4.4.14
```

Comment:
Isn't this implicitly `PROJECT_XYZ_TAG`? Could we get rid of this?

Reply:
Maybe, depends a bit on where the other conversations go.

Reply:
Specifically, we need to answer the question from my original comment first.